modules/sdk-core/src/bitgo/utils/tss/ecdsa/ecdsa.ts

```diff
@@ -469,14 +473,20 @@ export class EcdsaUtils extends baseTSSUtils<KeyShare> {
     let recipient: string;
     let keyShare: KeyShare;
     let otherShare: KeyShare;
+    let recipientGpgKey: openpgp.SerializedKeyPair<string>;
+    let senderGpgKey: openpgp.SerializedKeyPair<string>;
     if (recipientIndex === 1) {
       keyShare = userKeyShare;
       otherShare = backupKeyShare;
       recipient = 'user';
+      recipientGpgKey = userGpgKey;
+      senderGpgKey = backupGpgKey;
     } else if (recipientIndex === 2) {
       keyShare = backupKeyShare;
       otherShare = userKeyShare;
       recipient = 'backup';
+      recipientGpgKey = backupGpgKey;
+      senderGpgKey = backupGpgKey;
```

Comment:
why is the senderGpgKey always `backupGpgKey`? Isn't participantKeychain method used for both user/backup

Reply:
good catch